plat/hisilicon/poplar/bl2_plat_setup.c

```diff
@@ -9,11 +9,17 @@
 #include <bl_common.h>
 #include <console.h>
 #include <debug.h>
+#include <desc_image_load.h>
 #include <dw_mmc.h>
 #include <emmc.h>
 #include <errno.h>
 #include <generic_delay_timer.h>
 #include <mmio.h>
+#if LOAD_IMAGE_V2
```

Comment:
No need to guard #includes - the compiler will optimise these out if unused.

Reply:
ok, will remove